spec/factories/dishes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :dish do
+    sequence(:creator_id) { |n| }
```

Comment:
true, but for creating things for test set up (for example viewing a dish show page) you should have a creator in the test database so you can test things like seeing dish.creator